webapp/graphite/tags/utils.py

```diff
@@ -0,0 +1,94 @@
+"""Utility functions for tag databases."""
+import abc
+
+try:
+    from importlib import import_module
+except ImportError:  # python < 2.7 compatibility
+    from django.utils.importlib import import_module
+
+
+class TaggedSeries(object):
+  @classmethod
+  def parse(cls, path):
```

Comment:
on `render/functions.py:4082` this is called just to get the parsed `.tags` attribute. If `parse()` would simply return the tags, without instantiating the object, then it could be used without unnecessarily instantiating the object. Then it's `__init__` could simply `self.tags = self.parse(path)`